lib/index.js

```diff
@@ -79,19 +79,21 @@ internals.registrationConfig = (options) => {
     return config;
 };
 
-internals.initialize = function (server, next) {
+internals.initialize = (server, next) => {
 
-    const rootCollector = internals.state(server.root).collector;
+    const collector = internals.state(server.root).collector;
+    const rootKnex = Hoek.reach(collector, 'knexGroup.knex') || null;
```

Comment:
I might not be understanding it correctly, but Line 85 is reaching into collector for fullpath: `internals.state(server.root).collector.knexGroup.knex`, when it should omit the collector like internals.state(server.root).knexGroup.knex` right? I found this because I was trying to update a branch of [user-boilerplate](https://github.com/mattboutet/user-boilerplate) to use this branch of schwifty, and ran into some problems, it's weird that it passes all schwifty tests though, so I dunno what's going on. If I get it in a decent place that can focus on the problem, maybe I could push that code up and get some help in here, I'll link it if I can't figure it out